base/dcp_feed.go

```diff
@@ -250,7 +250,7 @@ func (r *DCPReceiver) GetMetaData(vbucketId uint16) (
 // RollbackEx should be called by cbdatasource - Rollback required to maintain the interface.  In the event
 // it's called, logs warning and does a hard reset on metadata for the vbucket
 func (r *DCPReceiver) Rollback(vbucketId uint16, rollbackSeq uint64) error {
-	Warn("DCP Rollback request.  Expected RollbackEx call - resetting vbucket %d to 0.", vbucketId)
+	WarnR("DCP Rollback request.  Expected RollbackEx call - resetting vbucket %s to 0.", MD(vbucketId))
```

Comment:
I don't think vbucketId needs to be wrapped, it's just an integer that's usually between 0 - 1024